src/System.IO.FileSystem/tests/FileSystemTest.cs

```diff
@@ -11,7 +11,9 @@ public abstract class FileSystemTest : IDisposable
     {
         public static readonly byte[] TestBuffer = { 0xBA, 0x5E, 0xBA, 0x11, 0xF0, 0x07, 0xBA, 0x11 };
 
-        public string TestDirectory { get; private set; }
+        private const bool CleanUp = true;
+
+        public readonly string TestDirectory;
```

Comment:
In the ctor, should these tests be created under Path.GetTempPath() rather than under Directory.GetCurrentDirectory()?  At least then any junk that accidentally gets left behind won't clutter up the tests directory.